magenta/models/rl_tuner/rl_tuner_test.py

```diff
@@ -0,0 +1,73 @@
+"""Tests for RLTuner and by proxy NoteRNNLoader.
+
+To run this code:
+$ bazel test rl_tuner:rl_tuner_test
+"""
+
+import os
+
+import tensorflow as tf
+import matplotlib
+matplotlib.use('Agg')
+
+import rl_tuner
+import rl_tuner_ops
+
+
+class RLTunerTest(tf.test.TestCase):
+
+  def setUp(self):
+    self.output_dir = '/tmp/rl_tuner_test'
```

Comment:
should use [mkdtemp](https://docs.python.org/2/library/tempfile.html#tempfile.mkdtemp)